hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -113,11 +115,14 @@ public void initialize(URI name, Configuration conf) throws IOException {
     String omPort = String.valueOf(-1);
     if (!isEmpty(remaining)) {
       String[] parts = remaining.split(":");
-      if (parts.length != 2) {
+      // Array length should only be 1 or 2
+      if (parts.length > 2) {
         throw new IllegalArgumentException(URI_EXCEPTION_TEXT);
       }
       omHost = parts[0];
-      omPort = parts[1];
+      // If port number is not specified, try default OM port
+      omPort = parts.length == 2 ?
+          parts[1] : String.valueOf(OZONE_OM_PORT_DEFAULT);
```

Comment:
Here do we need to use configuration.get(OZONE_OM_PORT, OZONE_OM_PORT_DEFAULT).
To support a case for reading default value from configuration.

Reply:
Updated. I'm using `OmUtils.getOmRpcPort(conf)` instead.